devbox.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jetify-com/devbox/main/.schema/devbox.schema.json",
+  "packages": ["nodejs@18.19.0"],
```

Comment:
Seems a bit old?

Reply:
Yeah, some projects are now on Node 20. When migrating projects to Devbox, I tend to leave the Node version as is. This ensures that my PR won't cause any errors.